core/src/main/java/org/apache/brooklyn/core/config/BasicConfigKey.java

```diff
@@ -365,6 +384,7 @@ public boolean isValueValid(T value) {
     }
 
     /** @see ConfigKey#getNameParts() */
+    @Deprecated
```

Comment:
That's included in the super-type `ConfigKey.getNameParts()`, which says `@deprecated since 0.12.0; use {@link #getName()}`